### PR TITLE
dev-python/flask-paginate: bumped to 0.5.5

### PR DESCRIPTION
**Pull request by florianjr**
Signed-off-by: Florian Förster <foerster.florianjr@gmail.com>
Closes: https://bugs.gentoo.org/717526
Package-Manager: Portage-3.0.0, Repoman-2.3.23

### CODE REVIEW

**Comment by florianjr** [comment]
Could I help maintaining this package furthermore?

**Comment by stkw0** [comment]
> Could I help maintaining this package furthermore?

I will be glad if you maintain this, maybe you can add yourself to the maintainers list. :)

**Comment by florianjr** [comment]
How can I do this? :)

**Comment by stkw0** [comment]
There is a file called metadata.xml with a list of maintainers. But I think before that a Gentoo developer should register you as a maintainer.

**Comment by florianjr** [comment]
Who should I contact? Then I can push it again with the right metadata.

**Comment by florianjr** [comment]
You mean as a proxy-maintainer?

**Comment by stkw0** [comment]
> You mean as a proxy-maintainer?

Yes. There is an IRC channel where you can ask.

**Comment by florianjr** [comment]
Here i added myself to the maintainers list, i think this should be enough.

**Comment by florianjr** [comment]
@stkw0 is this now enough to be merged?

**Comment by stkw0** [comment]
> @stkw0 is this now enough to be merged?

Version 0.7.0 is available, maybe we should bump to that version instead of 0.5.5. Also seems you forgot to update MY_COMMIT variable? It seems to be the same commit than in the older version.

**Comment by florianjr** [comment]
I’m kinda newbie, could you explain me the difference between $MY_COMMIT and $PV? I’ll do the bump later. Thanks for your attention :)

File: dev-python/flask-paginate/flask-paginate-0.5.5.ebuild [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+PYTHON_COMPAT=( pypy3 python3_{6,7} )
+
+inherit distutils-r1
+
+MY_COMMIT="510ad833106134711868653fb597bf75ea8ac34f"
+
+DESCRIPTION="Pagination support for flask"
+HOMEPAGE="https://flask-paginate.readthedocs.io"
+# https://github.com/lixxu/flask-paginate/issues/68
+SRC_URI="https://github.com/lixxu/flask-paginate/archive/${MY_COMMIT}.tar.gz -> ${P}.tar.gz"
```
Comment (jsmolic) [inline dev-python/flask-paginate/flask-paginate-0.5.5.ebuild:15]: `${MY_COMMIT}` is usually used when upstream doesn't use tags to mark releases, I checked now and they started using tags so for new version you can use `${PV}`  here and drop `${MY_COMMIT}`

**Comment by florianjr** [comment]
I’ve now bumped it again to newest version (0.6.0)

File: dev-python/flask-paginate/flask-paginate-0.6.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+PYTHON_COMPAT=( pypy3 python3_{6,7} )
+
+inherit distutils-r1
+
+MY_COMMIT="510ad833106134711868653fb597bf75ea8ac34f"
+
+DESCRIPTION="Pagination support for flask"
+HOMEPAGE="https://flask-paginate.readthedocs.io"
+# https://github.com/lixxu/flask-paginate/issues/68
+SRC_URI="https://github.com/lixxu/flask-paginate/archive/${MY_COMMIT}.tar.gz -> ${P}.tar.gz"
```
Comment (jsmolic) [inline dev-python/flask-paginate/flask-paginate-0.6.0.ebuild:15]: 0.7.0 is the latest version, and this is not correct, you should use `SRC_URI="https://github.com/lixxu/flask-paginate/archive/${PV}.tar.gz"`

File: dev-python/flask-paginate/flask-paginate-0.6.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( pypy3 python3_{6,7} )
+
+inherit distutils-r1
+
+MY_COMMIT="510ad833106134711868653fb597bf75ea8ac34f"
```
Comment (jsmolic) [inline dev-python/flask-paginate/flask-paginate-0.6.0.ebuild:10]: This commit is from 0.5.3 version (https://github.com/lixxu/flask-paginate/commit/510ad833106134711868653fb597bf75ea8ac34f)

File: dev-python/flask-paginate/flask-paginate-0.6.0.ebuild (hunk 2 above)
Comment (stkw0) [reply dev-python/flask-paginate/flask-paginate-0.6.0.ebuild:15]: Please remove that comment line also, now that the issue have been fixed it only disturbs :) 

Thank you for your effort.

**Comment by florianjr** [comment]
Ohhhh sorry. I’ll open a new and “clean” PR later, thanks for your help :)

**Comment by stkw0** [comment]
> Ohhhh sorry. I’ll open a new and “clean” PR later, thanks for your help :)

You don't need to do that. git rebase can do magic to fixup commits, remove commits and so ;) 
Check out https://wiki.gentoo.org/wiki/Gentoo_GitHub and more specifically  https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History

**Comment by florianjr** [comment]
That sounds good, thanks for giving me all this information to help me learn how to maintain ebuilds:)

**Pull request closed without merging**